pkg/influxunifi/influxdb.go

```diff
@@ -22,22 +23,35 @@ import (
 const PluginName = "influxdb"
 
 const (
-	defaultInterval   = 30 * time.Second
-	minimumInterval   = 10 * time.Second
-	defaultInfluxDB   = "unifi"
-	defaultInfluxUser = "unifipoller"
-	defaultInfluxURL  = "http://127.0.0.1:8086"
+	defaultInterval     = 30 * time.Second
+	minimumInterval     = 10 * time.Second
+	defaultInfluxDB     = "unifi"
+	defaultInfluxUser   = "unifipoller"
+	defaultInfluxOrg    = "unifi"
+	defaultInfluxBucket = "unifi"
+	defaultInfluxURL    = "http://127.0.0.1:8086"
 )
 
 // Config defines the data needed to store metrics in InfluxDB.
 type Config struct {
-	Interval  cnfg.Duration `json:"interval,omitempty" toml:"interval,omitempty" xml:"interval" yaml:"interval"`
-	URL       string        `json:"url,omitempty" toml:"url,omitempty" xml:"url" yaml:"url"`
-	User      string        `json:"user,omitempty" toml:"user,omitempty" xml:"user" yaml:"user"`
-	Pass      string        `json:"pass,omitempty" toml:"pass,omitempty" xml:"pass" yaml:"pass"`
-	DB        string        `json:"db,omitempty" toml:"db,omitempty" xml:"db" yaml:"db"`
-	Disable   bool          `json:"disable" toml:"disable" xml:"disable,attr" yaml:"disable"`
-	VerifySSL bool          `json:"verify_ssl" toml:"verify_ssl" xml:"verify_ssl" yaml:"verify_ssl"`
+	Interval cnfg.Duration `json:"interval,omitempty" toml:"interval,omitempty" xml:"interval" yaml:"interval"`
+	Version2 bool          `json:"version2,omitempty" toml:"version2,omitempty" xml:"version2" yaml:"version2"`
+
+	// V1
```

Comment:
The way godoc works, this sets the doc for `User` to `V1`.  I do not recommend a single comment like this for multiple struct members. Add a comment-per-struct member to produce beautiful code docs. 

Also, V2 doesn't use user/pass/db?

Reply:
Looks like the influx docs say auth token. Likely to be able to limit to write points only would be my guess. But I’ll admit I don’t use influx myself